### PR TITLE
Pod investigator enricher: move the reason-message part from enrinchments to description

### PR DESCRIPTION
**Pull request by RobertSzefler**

### CODE REVIEW

File: playbooks/robusta_playbooks/pod_investigator_enricher.py [hunk 1]
```diff
@@ -162,9 +162,17 @@ def report_pod_issue(
         event.add_enrichment(blocks)
 
     if reason:
+        # Update findings' descriptions.
         if message is None:
             message = "unknown"
-        event.add_enrichment([MarkdownBlock(f"\n\n{reason}: {message}")])
+        # The code below is essentially a specialized version of ExcutionBaseEvent.override_finding_attributes
```
Comment (arikalon1) [inline playbooks/robusta_playbooks/pod_investigator_enricher.py:168]: I would put this on the parent class, and a new method `extend_description` or `add_to_description`
`def add_to_description(addition: str)`

**Review by arikalon1 — COMMENTED** [review]
Looks good
Left one comment

File: playbooks/robusta_playbooks/pod_investigator_enricher.py (hunk 1 above)
Comment (RobertSzefler) [reply playbooks/robusta_playbooks/pod_investigator_enricher.py]: done

**Review by arikalon1 — APPROVED** [review]
LGTM

**Pull request merged**